app/resources/en/translation/screeners/starteligibility.json

```diff
@@ -3,7 +3,7 @@
   "header": "Apply for probate",
   "introduction": "Probate gives you the legal right to deal with a person&rsquo;s property, money and possessions (their &lsquo;estate&rsquo;) after they&rsquo;ve died.",
   "paragraph1": "You need to fill in the application online then send your documents to the probate registry. We&rsquo;ll tell you exactly what you need to send when you have completed your application.",
-  "paragraph2": "To apply in Welsh, download <a href=\"https://www.gov.uk/government/collections/ffurflenni-profiant\" target=\"_blank\" rel=\"noopener\">form PA1</a>. We&rsquo;re currently building a Welsh language online service.",
+  "paragraph2": "You can apply in Welsh by post. Use <a href=\"https://www.gov.uk/government/publications/form-pa1p-apply-for-probate-the-deceased-had-a-will\" target=\"_blank\" rel=\"noopener\">form PA1P</a> if the person who died left a will, and <a href=\"https://www.gov.uk/government/publications/form-pa1a-apply-for-probate-deceased-did-not-leave-a-will\" target=\"_blank\" rel=\"noopener\">form PA1A</a> if there is no will.",
```

Comment:
We have config vars for the links.
Use
`<a href=\"{applicationFormPA1P}" target=\"_blank\" rel=\"noopener\">form PA1P</a>`
and
`<a href=\"{applicationFormPA1A}" target=\"_blank\" rel=\"noopener\">form PA1A</a>`
instead...